doc/tutorial/install.rst

```diff
@@ -0,0 +1,48 @@
+Installation
+============
+
+All work will take place in a Python virtual environment.
+
+First, create a directory for your project, then a virtual environment within
+it, and activate the virtual environment::
+
+    mkdir rinoh-tutorial
+    cd rinoh-tutorial
+    python -m venv .env
+    source .env/bin/activate
+
+Install rinotype::
+
+    pip install rinohtype
+
+You'll notice that Sphinx and various other components are installed as
+dependencies of rinohtype.
+
+
+Start a new Sphinx project
+--------------------------
+
+Use ``sphinx-quickstart`` to bootstrap a new project::
+
+    sphinx-quickstart .
+
+You'll need to answer a few questions. Name the project ``All about me`` when
+asked. You can accept any defaults for other questions. Sphinx will create a
+set of files ready for you to start working with.
+
+Test your project, by running::
+
+    make html
```

Comment:
`sphinx-build -M html . _build` works as expected, but `sphinx-build -M rinoh . _build` gives me a:

```
Builder name rinoh not registered or available through entry point
```

for some not clear reason - because `make rinoh` works just fine.

Reply:
I'll leave it with ``make`` for now, and resolve this when I understand what's happening.